testsuite/standalone/src/test/java/org/jboss/as/test/integration/management/cli/LongOutputTestCase.java

```diff
@@ -297,4 +312,22 @@ private static void waitFor(Callable<Boolean> condition, long timeout) throws Ex
         Assert.fail("waitFor timed out");
     }
 
+
+    /**
+     * Check that the whole output was written at once when the output paging is disabled
+     */
+    @Test
+    public void testDisabledOutputPaging() throws Exception {
+        setupConsole(false);
+        consoleWriter.println("/:read-resource");
```

Comment:
You could perhaps use the elytron resource to be homogenous with the other testcase.